src/proto/grpc/testing/messages.proto

```diff
@@ -135,7 +144,7 @@ message StreamingOutputCallRequest {
   Payload payload = 3;
 
   // Whether to request the server to compress the response.
-  bool request_compressed_response = 6;
```

Comment:
woops, sorry, change made by mistake. I'm aware of the sanctity of tag numbers :)